autogen/agentchat/groupchat.py

```diff
@@ -102,6 +102,8 @@ def custom_speaker_selection_func(
 
     agents: List[Agent]
     messages: List[Dict]
+    select_speaker_auto_llm_config: Optional[Union[Dict, Literal[False]]] = None
```

Comment:
Thanks for changing the names, would you be able to move these down further and put them with the other select_speaker attributes, I think around `select_speaker_auto_verbose: Optional[bool] = False` would be good. Sorry :)